libraries/standard/mqtt/src/mqtt.c

```diff
@@ -1769,25 +1770,23 @@ MQTTStatus_t MQTT_ProcessLoop( MQTTContext_t * pContext,
                                uint32_t timeoutMs )
 {
     MQTTStatus_t status = MQTTBadParameter;
-    MQTTGetCurrentTimeFunc_t getTimeStampMs = NULL;
     uint32_t entryTimeMs = 0U, remainingTimeMs = timeoutMs, elapsedTimeMs = 0U;
 
     if( pContext == NULL )
     {
-        LogError( ( "MQTT Context cannot be NULL." ) );
+        LogError( ( "Invalid input parameter: MQTT Context cannot be NULL." ) );
     }
-    else if( pContext->callbacks.getTime == NULL )
+    else if( pContext->getTime == NULL )
     {
-        LogError( ( "MQTT Context must set callbacks.getTime." ) );
+        LogError( ( "Invalid input parameter: MQTT Context must have valid getTime." ) );
     }
     else if( pContext->networkBuffer.pBuffer == NULL )
     {
-        LogError( ( "The MQTT context's networkBuffer must not be NULL." ) );
+        LogError( ( "Invalid input parameter: The MQTT context's networkBuffer must not be NULL." ) );
     }
     else
     {
-        getTimeStampMs = pContext->callbacks.getTime;
-        entryTimeMs = getTimeStampMs();
+        entryTimeMs = pContext->getTime();
```

Comment:
I think it's odd you changed this for the process loop but not the receive loop. Why not keep them similar?

Reply:
Thanks for catching. Made the two consistent